metadata-ingestion/src/datahub/ingestion/run/classifier_pipeline.py

```diff
@@ -0,0 +1,252 @@
+        now = datetime.datetime.now()
+
+        are_writes_successful = True
+        for col_name, class_result in classification_result.items():
+            for law in class_result.privacy_laws:
+                are_writes_successful &= self._attach_glossary_terms(urn, col_name, "PrivacyLaw.{}".format(law))
+            for type in class_result.pii_types:
+                are_writes_successful &= self._attach_glossary_terms(urn, col_name, "PiiData.{}".format(type))
+
+        if are_writes_successful:
+            if is_first_time_classified:
+                self.pii_classification_state_table.put_item(
+                    Item={
+                            'pk': urn,
+                            'sk': self.DATASET_RECORD_SK,
+                            'createdDate': now.isoformat(),
+                            'lastEvalDate': now.isoformat()
+                        }
+                    )
+            else:
+                self.pii_classification_state_table.update_item(
+                    Key={
+                        'pk': urn,
+                        'sk': self.DATASET_RECORD_SK
+                    },
+                    UpdateExpression="SET lastEvalDate = :newEvalDate",
+                    ExpressionAttributeValues={":newEvalDate": now.isoformat()}
+                )
+
+    def _attach_glossary_terms(self, urn: str, subfield: str, glossary_term: str) -> bool:
+
+        url = "{}{}".format(self.datahub_base_url, self.DATAHUB_GRAPHQL_ENDPOINT)
+
+        req = {
+            "operationName": "addTerm",
+            "variables": {
+                "input": {
+                    "termUrn": "urn:li:glossaryTerm:{}".format(glossary_term),
+                    "resourceUrn": urn,
+                    "subResource": subfield,
+                    "subResourceType": "DATASET_FIELD"
+                }
+            },
+            "query": "mutation addTerm($input: TermAssociationInput!) {\n  addTerm(input: $input)\n}\n"
+        }
+
+        r = requests.post(url, json=req, cookies=self.datahub_cookies)
+
+        success = "addTerm" in r.text
+
+        if success:
+            logger.info("Attached term {} to {} on field {} with code {}".format(glossary_term, urn, subfield, str(r.status_code)))
+        else:
+            logger.error("Failed to attach term {} to {} on field {} with code {}".format(glossary_term, urn, subfield, str(r.status_code)))
+
+        return success
+
+    @classmethod
+    def create(cls, config_dict: dict) -> "Pipeline":
+        config = ClassifierPipelineConfig.parse_obj(config_dict)
+        return cls(config)
+
+    def run(self) -> None:
+        extractor: Extractor = self.extractor_class()
+        for wu in self.source.get_workunits():
+            extractor.configure({}, self.ctx)
+
+            for record_envelope in self.transform(extractor.get_records(wu)):
```

Comment:
I'm interested in what prod runtimes would look like as-is (single-threaded). I think there's a good chance the time/headache of making this multi-threaded would be worth it, but I'll defer that for now